protos/robot/NUgus.proto

```diff
@@ -7,7 +7,7 @@ PROTO NUgus [
   field  SFVec3f     translation          0 0 0
   field  SFRotation  rotation             0 1 0 0
   field  SFString    name                 "NUgus"            # Is `Robot.name`.
-  field  SFString    controller           "void"             # Is `Robot.controller`.
+  field  SFString    controller           "player"             # Is `Robot.controller`.
```

Comment:
Should this be `nugus_controller`?

Reply:
https://github.com/RoboCup-Humanoid-TC/webots/tree/release/projects/samples/contests/robocup/controllers/player

Reply:
Can this go in a comment in the file?